Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    sfctl (0.1.0)
+    sfctl (1.0.0)
```

Comment:
"master" already contains version "1.0.1". Please recheck if you need to update it.